src/EA.Weee.Web/Services/OrganisationTransactionService.cs

```diff
@@ -37,6 +37,12 @@ public async Task CaptureData<T>(string accessToken, T model) where T : class
                     case PreviousRegistrationViewModel previousRegistrationModel:
                         transaction.PreviousRegistration = previousRegistrationModel.SelectedValue.GetValueFromDisplayName<YesNoType>();
                         break;
+                    case RepresentingCompanyDetailsViewModel representingCompanyDetailsViewModel:
```

Comment:
Dont clear the other fields here as they are used by other pages,

Reply:
updated